app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -95,9 +96,17 @@ export default function Overview({ user, setIsRequesting }: OverviewProps) {
         ) : (
           <>
             <Button onClick={() => setIsRequesting(true)}>{REQUEST}</Button>
-            {user.friends !== User.FriendshipStatus.FRIENDS ? (
-              <AddFriendButton
-                isPending={user.friends === User.FriendshipStatus.PENDING}
+            {user.friends !== User.FriendshipStatus.FRIENDS &&
+              user.friends !== User.FriendshipStatus.PENDING && (
+                <AddFriendButton
+                  userId={user.userId}
+                  setMutationError={setMutationError}
+                />
+              )}
```

Comment:
```suggestion
            {user.friends === User.FriendshipStatus.NOT_FRIENDS && (
                <AddFriendButton
                  userId={user.userId}
                  setMutationError={setMutationError}
                />
             )}
```
Since FriendshipStatus is an enum and this is the other possible value, you can check the user is `NOT_FRIENDS` instead to simplify the logic